boltdb/version.go

```diff
@@ -0,0 +1,80 @@
+package boltdb
+
+import (
+	"bytes"
+	"context"
+
+	bolt "go.etcd.io/bbolt"
+)
+
+func updateVersion(
+	ctx context.Context,
+	tx *bolt.Tx,
+	h string,
```

Comment:
Is this the handler key, if so, perhaps `hk` as the name?
I feel that these functions need to be documented as well.

Reply:
have been fixed in the latest commits.